cmd/notify-webhook.go

```diff
@@ -107,8 +107,8 @@ func lookupEndpoint(urlStr string) error {
 		},
 	}
 
-	// Set content-type.
-	req.Header.Set("Content-Type", "application/json")
+	// Set content-length to zero as there is no payload.
+	req.Header.Set("Content-Length", "0")
```

Comment:
we don't need to set like this explicitly. just set it as `req.ContentLength = 0`

Reply:
done